patient-appointment-api.yml

```diff
@@ -903,6 +969,69 @@ components:
           items:
             $ref: "#/components/schemas/ContactPoint"
       description: Patient to be processed
+    PatientRecord:
+      required:
+        - patient
+        - responsibleProvider
+      type: object
+      properties:
+        patient:
+          $ref: "#/components/schemas/Patient"
+        responsibleProvider:
+          description: "Provider responsible for this patient's care. This can be\
+            \ the Primary Care Physician (PCP), most recent provider, or a provider\
+            \ assigned for future care."
+          allOf:
+            - $ref: "#/components/schemas/Provider"
+        activeIcd10Diagnoses:
+          type: array
+          description: All active diagnoses (problems) known about the patient. Must
+            all be ICD-10 codes. If the patient is an existing patient then this list
+            should be populated with all known diagnoses
+          items:
+            $ref: "#/components/schemas/Diagnosis"
+        activeIcd9Diagnoses:
+          type: array
+          description: All active diagnoses (problems) known about the patient. Must
+            all be ICD-9 codes. If the patient is an existing patient then this list
+            should be populated with all known diagnoses. **ICD-10 should be greatly
+            preferred if available.**
+          example:
+            - code: "401.1"
+              displayName: Benign essential hypertension
+              date: 2023-12-12
+          items:
+            $ref: "#/components/schemas/Diagnosis"
+        activeSnomedDiagnoses:
+          type: array
+          description: All active diagnoses (problems) known about the patient. Must
+            all be SNOMED codes. If the patient is an existing patient then this list
+            should be populated with all known diagnoses. **ICD-10 should be greatly
+            preferred if available.**
+          example:
+            - code: "34713006"
+              displayName: Vitamin D deficiency
+              date: 2023-11-12
```

Comment:
I think all dates like this need to be manually quoted or it doesn't look correct when deployed to Mintlify. Could test it out but I think that's still the case. I can't get MN to do the quoting for us.

Reply:
Argh, missed this one, thanks

Reply:
two, actually. done!